org.jacoco.doc/docroot/doc/changes.html

```diff
@@ -38,6 +38,14 @@ <h3>Fixed Bugs</h3>
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/672">#672</a>).</li>
 </ul>
 
+<h3>API Changes</h3>
+<ul>
+  <li>The XML report now has an optional attribute <code>sourcefilename</code>
+      on the <code>class</code> element to allow unambiguously relate classes
+      to source files. The JaCoCo DTD version has been updated to 1.1. 
```

Comment:
Currently existing items consisting of multiple sentences use only one dot after link to GitHub in the last sentence.

Reply:
Thx, fixed.